src/blocks/button/Button.utils.ts

```diff
@@ -26,6 +28,11 @@ export const getButtonVariantStyles = (variant: ButtonVariant) => {
           background-color: var(--components-button-primary-background-disabled);
           color: var(--components-button-primary-text-disabled);
         }
+
+        [role='spinner'] {
+          border-color: var(--components-button-primary-icon-default) transparent transparent transparent;
```

Comment:
```suggestion
          border-color: var(--components-button-primary-icon-default) var(--surface-transparent) var(--surface-transparent) var(--surface-transparent);
```

Also see if there is a way to only provide top border color.